test/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/DeclarationXml_GovernmentAgencyGoodsItemSpec.scala

```diff
@@ -27,6 +27,7 @@ class DeclarationXml_GovernmentAgencyGoodsItemSpec extends WordSpec with MustMat
     "be populated in the XML" when {
       "the item level data is present in the Declaration" in {
         val governmentAgencyGoodsItem = GovernmentAgencyGoodsItem(
+          additionalDocuments = Seq.empty,
```

Comment:
do we need any tests for the resulting `<AdditionalDocument>` elements?  or are they covered by the original viewmodel-centric tests?

Reply:
Covered by the XmlWriter_AdditionalDocumentSpec, WriteOffXmlWriterSpec and SubmitterXmlWriterSpec I think.